### PR TITLE
Updated datasets for this test

### PR DESCRIPTION
**Pull request by stsci-hack**
Changes to correctly handle different instruments reference files and to update the test datasets to ones which more easily demonstrate the effects of the a priori solutions.  

These changes should address the problems seen in the regression tests with failures due to reference file problems.

### CODE REVIEW

**Comment by stsci-hack** [comment]
The code for this test has been reorganized to share the comparison function between 2 instrument-specific classes.

File: tests/hla/test_apriori.py [hunk 1]
```diff
@@ -6,91 +6,102 @@
...
+
+        # Check that rotation and scale are within
+        delta_rot = np.abs(results['rotation'] - pipeline_results['rotation']) / pipeline_results['rotation']
+        delta_scale = np.abs(results['scale'] - pipeline_results['scale']) / pipeline_results['scale']
+        rot = (delta_rot < limit).all()
+        scale = (delta_scale < limit).all()
+
+        # Determine success/failure of this dataset's fit
+        wcs_success = status and fit_qual and delta and rot and scale
+        if wcs_success:
```
Comment (mcara) [inline tests/hla/test_apriori.py]:
```python
if all([status, fit_qual, delta, rot, scale]):
```
If accepted, make sure to delete previous line.

**Comment by stsci-hack** [comment]
All review comments have been implemented in 4c91ec6.

**Comment by stsci-hack** [comment]
If it is any relief, the updated version of this test passes when run interactively. ;-)

**Pull request merged**